lib/engine/step/g_18_ms/track.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require_relative '../track'
+
+module Engine
+  module Step
+    module G18MS
+      class Track < Track
+        ACTION_HEX_OF_INTEREST = 'B10'
+        BONUS_HEX = 'B12'
+
+        def lay_tile(action, extra_cost: 0, entity: nil)
+          entity ||= action.entity
+          super
+
+          return if action.tile.hex.name != ACTION_HEX_OF_INTEREST ||
+            @game.chattanooga_reached ||
+            !@game.loading ||
```

Comment:
this seems wrong. this means there will be different behavior

Reply:
I have made a fix. Better now?